src/python/WMCore/Services/WMArchive/DataMap.py

```diff
@@ -110,6 +111,12 @@
         # "GUID": '',
         # 'StageOutCommand': ''
     }],
+    'WMCMSSWSubprocess': {
```

Comment:
I feel like we should not be defining any default for these metrics, as it does not give a clear picture of whether it reads 0 or it is just the default value.
Is there any strong reason to set these defaults? Note that other attributes do not have any default!

Reply:
ok, I removed defaults, usually I prefer to have defaults in place but it seems that they can be easily skipped if not provided and therefore not having them is ok too.